$MirOS: ports/print/cups/patches/patch-locale_Makefile,v 1.2 2006/11/08 18:25:45 bsiegert Exp $
--- locale/Makefile.orig	Fri Sep 28 20:30:08 2007
+++ locale/Makefile	Sun Oct 26 22:56:10 2008
@@ -103,8 +103,8 @@ pot:
 #
 
 checkpo:	checkpo.o ../cups/$(LIBCUPS)
-	echo Linking $<...
-	$(CC) $(LDFLAGS) -o checkpo checkpo.o $(LIBS)
+	echo Linking $@...
+	$(LIBTOOL) --mode=link $(CC) $(LDFLAGS) -o checkpo checkpo.o $(LIBS)
 
 
 #
@@ -115,8 +115,8 @@ checkpo:	checkpo.o ../cups/$(LIBCUPS)
 #
 
 po2strings:	po2strings.o ../cups/$(LIBCUPS)
-	echo Linking $<...
-	$(CC) $(LDFLAGS) -o po2strings po2strings.o $(LIBS)
+	echo Linking $@...
+	$(LIBTOOL) --mode=link $(CC) $(LDFLAGS) -o po2strings po2strings.o $(LIBS)
 
 
 #
@@ -127,8 +127,8 @@ po2strings:	po2strings.o ../cups/$(LIBCU
 #
 
 strings2po:	strings2po.o
-	echo Linking $<...
-	$(CC) $(LDFLAGS) -o strings2po strings2po.o
+	echo Linking $@...
+	$(LIBTOOL) --mode=link $(CC) $(LDFLAGS) -o strings2po strings2po.o
 
 
 #
@@ -139,8 +139,8 @@ strings2po:	strings2po.o
 #
 
 translate:	translate.o ../cups/$(LIBCUPS)
-	echo Linking $<...
-	$(CC) $(LDFLAGS) -o translate translate.o $(LIBS)
+	echo Linking $@...
+	$(LIBTOOL) --mode=link $(CC) $(LDFLAGS) -o translate translate.o $(LIBS)
 
 
 #
